### PR TITLE
Ameen

### PR DESCRIPTION
**Pull request by AmeenShaikh025**
Implemented throttle using underscore js

### CODE REVIEW

File: explore.html [hunk 1]
```diff
@@ -511,6 +511,7 @@ <h2>support</h2>
         </div>
     </footer>
 
+    <script src="https://cdn.jsdelivr.net/npm/underscore@1.12.0/underscore-min.js"></script>
```
Comment (muskeinsingh) [inline explore.html:514]: what is the size impact of this? can we do better?

File: js/home.js [hunk 2]
```diff
@@ -23,21 +37,23 @@ function scrollDetect(){
...
             nav.classList.add("hide__nav");
             nav.classList.remove("show__nav");
-        }else{
+        }
+        else{
             lastScroll = currentScroll;
-            //console.log("Scrolling UP");
-            nav.classList.add("show__nav");
-            nav.classList.remove("hide__nav");
+                lastScroll = currentScroll;
```
Comment (muskeinsingh) [inline js/home.js:42]: repetition

File: explore.html (hunk 1 above)
Comment (muskeinsingh) [reply explore.html:514]: let's add lighthouse scores on each PR

**Review by muskeinsingh — COMMENTED** [review]
comments

File: explore.html (hunk 1 above)
Comment (AmeenShaikh025) [reply explore.html:514]: 1. working on compressing the image, optimizing code, and focusing on the lighthouse report.
2. Added lighthouse to the repo created GitHub actions and status

**Comment by AmeenShaikh025** [comment]
Added lighthouse

**Pull request merged**